src/NUnitFramework/framework/Internal/Execution/CompositeWorkItem.cs

```diff
@@ -86,7 +86,7 @@ protected override void PerformWork()
                             Result.SetResult(ResultState.Success);
 
                             if (Children.Count > 0)
-                            {
+                            {   
```

Comment:
Whoops! Didn't see that. 😊 I'll wait to see if there are any more serious issues.